File: server/main/configs/keys.js
const Master = require('../models/master');


module.exports = {
    getMaster: async function (key){
        var masterKey = {};
        try{
            masterKey = await Master.findOne({ key });
            if (!masterKey) throw new Error("Master key not found");
        } catch(e){
            console.error(e);
            masterKey = { value: null}
        } finally{
            return masterKey.value || null;
        }
    }
}

File: server/usersManager/config/keys.py
from models.master import Master

from config.mongo import init_db, close_db

def getMaster(key):
    try:
        init_db()
        master = Master.objects.get(key=key)
        close_db()
        return master.value
    except (Exception):
        return None

MASTER_KEY = getMaster('MASTER_KEY')